File: xamarin/Application.XForms/Application.XForms/Models/Category.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Application.XForms.Models
{
    /// <summary>
    /// Category, data model for communication between CRUDView and CRUDController (CRUD2CRUD).
    /// </summary>
    public class Category
    {
        public Guid Uid { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public byte[] RowVersion { get; set; }
    }
}

File: xamarin/Application.XForms.Service/wwwroot/js/Models/Category.js
function Category(data) {
    var instance = this;

    //required for row uniqueness 
    instance._datakey = data.Uid;

    instance.Uid = data.Uid;

    instance.Title = ko.observable(data.Title);
    instance.Title.extend({
        required: true,
        maxLength: 100
    });

    instance.Description = ko.observable(data.Description);
    instance.Description.extend({
        maxLength: 200
    });

    instance.RowVersion = data.RowVersion;
}